redisCredentials.js

```diff
@@ -0,0 +1,4 @@
+const REDIS_PORT = 6379;
```

Comment:
would it be a good idea to add this in env variable rather than hardcoding it here.

Reply:
Its not a secret or a variable that we would like to change based on production environment, so I don't think it should go in the .env file